whylog/config/investigation_plan.py

```diff
@@ -21,6 +21,13 @@ def __init__(self, suspected_rules, investigation_metadata, effect_clues):
     def get_next_investigation_step_with_log_type(self):
         return (meta_data for meta_data in self._investigation_metadata)
 
+    def suspected_rules(self):
+        for rule in self._suspected_rules:
+            yield rule
+
+    def get_effect_clues(self):
+        return self._effect_clues
```

Comment:
this class only has 3 fields, all publicly readable, so maybe it should just use `@property` instead of java-like hand-crafted getters